Classes/Controller/ProductAreaController.php

```diff
@@ -42,6 +43,9 @@ public function injectProductAreaRepository(\NITSAN\NsT3dev\Domain\Repository\Pr
      */
     public function listAction(): \Psr\Http\Message\ResponseInterface
     {
+        $this->eventDispatcher->dispatch(
+            new FrentendRendringEvent()
```

Comment:
Check spelling of FrentendRenderingEvent(), it should be FrontendRenderingEvent().